src/communication/websocket/auth.hpp

```diff
@@ -19,15 +19,24 @@
 
 namespace communication::websocket {
 
-class SafeAuth {
+class IAuthentication {
```

Comment:
IAuthentication? What's the `I`? :D

Reply:
Since this is a pure abstract class maybe you can name it as `AuthenticatioInterface`

Reply:
I stands for the interface, but I think your point is valid I am looking and haven't seen this naming convention anywhere in codebase